### PR TITLE
Fix kickstart install with infiniband (#1177032)

### PR DESCRIPTION
**Pull request by jkonecny12**
Split infiniband from "802-3-ethernet" with different NetworkManager
settings.

Resolves: rhbz#1177032

### CODE REVIEW

File: pyanaconda/network.py [hunk 1]
```diff
@@ -965,7 +975,10 @@ def ks_spec_to_device_name(ksspec=""):
         # "XX:XX:XX:XX:XX:XX" (mac address)
         elif ':' in ksspec:
             try:
-                hwaddr = nm.nm_device_perm_hwaddress(dev)
+                if nm.nm_device_type_is_infiniband(dev):
+                    hwaddr = nm.nm_device_hwaddress(dev)
+                else:
+                    hwaddr = nm.nm_device_perm_hwaddress(dev)
```
Comment (vpodzime) [inline pyanaconda/network.py]: This deserves a comment.

**Comment by vpodzime** [comment]
Looks good to me other than the suggestion above.

**Comment by jkonecny12** [comment]
I changed this patch after @vpodzime comment. Thank you for the idea.
The PR is quite different now.

**Comment by vpodzime** [comment]
Nice, thanks! Looks great to me now.

**Comment by jkonecny12** [comment]
Pushed

**Pull request closed without merging**